cmd/openshift/openshift.go

```diff
@@ -13,7 +13,9 @@ func main() {
 	defer serviceability.BehaviorOnPanic(os.Getenv("OPENSHIFT_ON_PANIC"))()
 	defer serviceability.Profile(os.Getenv("OPENSHIFT_PROFILE")).Stop()
 
-	runtime.GOMAXPROCS(runtime.NumCPU())
+	if len(os.Getenv("GOMAXPROCS")) == 0 {
+		runtime.GOMAXPROCS(runtime.NumCPU())
```

Comment:
The scheduler controls it, and ultimately it only uses multiple cores when it truly needs it (the scheduler biases to time slicing on extra cores).  Grabbing more cores than we need is something an admin now has the leverage to control via gomaxprocs 

> On Jun 9, 2015, at 11:11 PM, Jordan Liggitt <notifications@github.com> wrote:
> 
> In cmd/openshift/openshift.go:
> 
> > @@ -13,7 +13,9 @@ func main() {
> >  	defer serviceability.BehaviorOnPanic(os.Getenv("OPENSHIFT_ON_PANIC"))()
> >  	defer serviceability.Profile(os.Getenv("OPENSHIFT_PROFILE")).Stop()
> >  
> > -	runtime.GOMAXPROCS(runtime.NumCPU())
> > +	if len(os.Getenv("GOMAXPROCS")) == 0 {
> > +		runtime.GOMAXPROCS(runtime.NumCPU())
> not sure grabbing all cores by default is a good idea
> 
> —
> Reply to this email directly or view it on GitHub.
>